package-lock.json

```diff
@@ -3621,6 +3621,29 @@
         "@bbc/psammead-styles": "^4.3.0"
       }
     },
+    "@bbc/psammead-radio-schedule": {
+      "version": "0.1.0-alpha.19",
+      "resolved": "https://registry.npmjs.org/@bbc/psammead-radio-schedule/-/psammead-radio-schedule-0.1.0-alpha.19.tgz",
+      "integrity": "sha512-a4V6q3XLtzDFRV6CablHEPuJdY8HNCHwnGUOQjcN1Qj4rbJWLrSIl+kbcOnGDfacZYxkgWdKZMxBgNRB0NCPzw==",
+      "requires": {
+        "@bbc/gel-foundations": "^4.0.1",
+        "@bbc/psammead-assets": "^2.13.0",
+        "@bbc/psammead-styles": "^4.3.0",
+        "@bbc/psammead-timestamp-container": "^2.7.7"
+      },
+      "dependencies": {
```

Comment:
Could you remove this dependencies block and make sure the component has the same timestamp container dependency as simorgh? :)

Reply:
But since in Psammead `radio-schedule` uses newer version (`^2.7.7`) of `timestamp-container`, would it not cause problems if I downgrade it in Simorgh?

Reply:
Hey! I think Simorgh can be bumped to `^2.7.7` safely as it's just a patch :)

Reply:
Bumped up `timestamp-container` now :)